apps/src/schoolInfo/utils/buildSchoolData.ts

```diff
@@ -38,25 +34,17 @@ export function buildSchoolData({
     schoolId === NonSchoolOptions.NO_SCHOOL_SETTING
   ) {
     return {
-      user: {
```

Comment:
Can you explain why this change is necessary? I have a guess but I'm not sure!

Reply:
when I first created the `buildSchoolData` helper function, I thought it was only going to be used with `PATCH /api/v1/user_school_infos` where the params are extracted in the `user_school_infos_controller`. un-nesting the attributes made it reusable in the `POST /api/v1/pd/workshop/:workshopId/enrollments` route

Reply:
Is this function not used in sign up?

Reply:
it's not, only in the `updateSchoolInfo` helper method that calls the api